ada_feeding/ada_feeding/visitors/moveto_visitor.py

```diff
@@ -153,20 +152,7 @@ def run(self, behaviour: py_trees.behaviour.Behaviour) -> None:
         if self.start_time is None:
             self.start_time = self.node.get_clock().now()
 
-        if isinstance(behaviour, ada_feeding.behaviors.MoveTo):
-            # If in MoveTo action, copy from there
-            self.feedback.motion_initial_distance = (
-                behaviour.tree_blackboard.motion_initial_distance
-            )
-            self.feedback.motion_curr_distance = (
-                behaviour.tree_blackboard.motion_curr_distance
-            )
-
-            # Check for flip between planning/motion
-            if behaviour.tree_blackboard.is_planning != self.feedback.is_planning:
-                self.start_time = self.node.get_clock().now()
-                self.feedback.is_planning = behaviour.tree_blackboard.is_planning
-        elif (
+        if (
```

Comment:
Note to @egordon for #122 
It is best to create a ServoMove Behavior (which takes in a twist and duration) so that this Visitor can capture that as part of the feedback (currently it is treated as "planning", which is incorrect).